cmd/update-notifier.go

```diff
@@ -28,25 +28,39 @@ import (
 	"github.com/fatih/color"
 )
 
+// computeUpdateMessage - calculates the update message, only if a
+// newer version is available.
+func computeUpdateMessage(downloadURL string, older time.Duration) string {
+	if downloadURL == "" || older <= 0 {
+		return ""
+	}
+
+	// Compute friendly duration string to indicate time
+	// difference between newer and current release.
+	t := time.Now()
```

Comment:
It is to be consistent with our internal usage of time. We only deal with UTC everywhere.. @balamurugana